lib/internal/Magento/Framework/Stdlib/DateTime/Filter/DateTime.php

```diff
@@ -42,7 +42,7 @@ public function __construct(\Magento\Framework\Stdlib\DateTime\TimezoneInterface
     public function filter($value)
     {
         try {
-            $dateTime = $this->_localeDate->date($value, null, false);
+            $dateTime = $this->_localeDate->date($value, null, false, false);
```

Comment:
I don't think passing `$includeTime = false` is correct for `DateTime` filter.

I see such `false` correctly passed in `\Magento\Framework\Stdlib\DateTime\Filter\Date::filter` though. Should we replace `DateTime` with `Date` filter maybe?

Reply:
\Magento\Framework\Stdlib\DateTime\Filter\Date::filter  was edited before with added passing "false" there.
its passed through Magento\Framework\Stdlib\DateTime;:date where posting "$includeTime = true" will lead to wrong $timeTypein row 170

Also : DateTime is extending Magento\Framework\Stdlib\DateTime\Filter\Date with changing __construct

So i think changing to "Date" would not be the correct way and the edit for "filter" was forgotten in extended?

Reply:
According to my understanding if our `DateTime` filter will ignore time it will be pretty useless :) So I believe just in place which you try to fix wrong filter class is used.

Reply:
So you mean, removing the "public function filter($value)" from "DateTime" and keeping the constructor? Or what do you mean ? ^^